bootstrap/v2/pkg/kfapp/existing_arrikto/existing.go

```diff
@@ -174,6 +185,13 @@ func (existing *Existing) Apply(resources kftypes.ResourceEnum) error {
 		return internalError(errors.WithStack(err))
 	}
 
+	//for dex the AuthServiceClientSecret shouldn't be base64 encoded
+	authServiceClientSecretDecoded, err := base64.StdEncoding.DecodeString(data.AuthServiceClientSecret)
```

Comment:
It seems that we need this because Dex doesn't support setting the client id with secrets.
https://github.com/dexidp/dex/issues/1099

Because of this, we have to decode the value and set it directly in the Dex ConfigMap.
Would be nice if we could solve this upstream or via an initContainer.
@ryandawsonuk can you add a TODO comment here explaining why we do this ?